Packs/CommonScripts/Scripts/TextFromHTML/TextFromHTML.py

```diff
@@ -5,26 +5,57 @@
 import re
 
 
-def get_plain_text(html_regex):
+def get_plain_text(html: str):
     data = ''
-    if html_regex and html_regex.group(0):
-        data = re.sub(r'<.*?>', '', html_regex.group(0))
+    if html:
+        data = re.sub(r'<br>', '\n', html, flags=re.M + re.S + re.I + re.U)
+        data = re.sub(r'<.*?>', '', html, flags=re.M + re.S + re.I + re.U)
         entities = {'quot': '"', 'amp': '&', 'apos': "'", 'lt': '<', 'gt': '>', 'nbsp': ' ',
                     'copy': '(C)', 'reg': '(R)', 'tilde': '~', 'ldquo': '"', 'rdquo': '"', 'hellip': '...'}
         for e in entities:
             data = data.replace(f'&{e};', entities[e])
+        data = re.sub(r'[ \t]{2,}', ' ', data)
+        data = re.sub(r'(\r?\n){3,}', '\n\n', data)
+        data = re.sub(r'\s+$', '', data)
     return data
 
 
-def text_from_html(args):
-    html = args['html']
+def get_body(html: str, html_tag: str, allow_fallback: bool = False):
+    if html and html_tag:
+        body = re.search(fr'<{html_tag}.*/{html_tag}>', html, re.M + re.S + re.I + re.U)
+
+        if body and body.group(0):
+            return body.group(0)
+        elif allow_fallback and html_tag.lower() == 'body':
+            return html
+    return ''
+
+
+def text_from_html(args: dict):
+    html = args.get('html', '')
```

Comment:
Please revert this. "html" is a required parameter that should always be assigned.
We want it to fail if it doesn't exist for some reason (and not assign it an empty string value)/

```suggestion
    html = args['html']
```

Reply:
This only was changed to address the demisto-sdk linter warning: 
W9019 text_from_html: It is best practice to use .get when accessing the arg/params dict object rather then direct access.
I'll revert the change

Reply:
This warning is incorrect (it's not "best practice"), I raised an internal issue to remove it.